src/OrchardCore.Modules/OrchardCore.Spatial/Views/GeoPointField.cshtml

```diff
@@ -0,0 +1,11 @@
+@model DisplayGeoPointFieldViewModel
+@using OrchardCore.Mvc.Utilities;
+@using OrchardCore.Spatial.ViewModels
+
+@{
+    var name = (Model.PartFieldDefinition.PartDefinition.Name + "-" + Model.PartFieldDefinition.Name).HtmlClassify();
+}
+
+<div class="field field-type-booleanfield field-name-@name">
```

Comment:
booleanfield => geopointfield

Should we display a map?

Reply:
I think there is one displayed after you entered the Lat and Long values.

Reply:
![image](https://user-images.githubusercontent.com/3228637/112540206-47868900-8d88-11eb-80c2-a49ede65abf7.png)

Reply:
You are showing me the editor (GeoPointField-Leaflet.Edit.cshtml , which lacks an eol btw).
This is the Display of the field.

Reply:
I'm not sure if we want the display of these to be bound by default to a javascript dependency "leafleft.js".
By default, it should display the coordinates because that's what it is basically.